### DISCUSSION TITLE
Offending chibi Monika?

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#8934] [QA - tech support + general]
In the letter that chibi Monika sent me, it was written that I would not tell monica about her, but I tried to tell and got scared (due to an unexpected glitch and chibi Monika), now the question. Will chibi Monika be offended by my attempt? She said not to talk about her but I said. Stupid question, but I want to make sure everything goes well.

**Comment by hlepnamingblyat** [comment]
I believe so.
Just let it be a surprise but most importantly, trust her.

**Comment by Nikv2005** [comment]
> I believe so.
> Just let it be a surprise but most importantly, trust her.

Chibi Monika is the same as big Monika and i trust her. But I wanted to try it out of curiosity, I hope that she is not particularly disappointed in me.
